src/cljs/athens/page.cljs

```diff
@@ -4,6 +4,10 @@
             #_[reitit.frontend.easy :as rfee]
             #_[reagent.core :as reagent]))
 
+
+(defn on-block-click [uid]
+  (dispatch [:navigate :page {:id uid}]))
```

Comment:
This seems like a really helpful function. I would move it to `router.cljs` and refactor any other occurrences.

Reply:
Ok sounds good!